tests/validation/converters/test_pandapower_converter_output.py

```diff
@@ -92,8 +102,10 @@ def test_generate_output():  # TODO: REMOVE THIS FUNCTION
     assert_valid_input_data(input_data=input_data)
     pgm = PowerGridModel(input_data=input_data)
     output_data = pgm.calculate_power_flow()
-    json_converter = PgmJsonConverter(destination_file=PGM_OUTPUT_FILE)
-    json_converter.save(data=output_data, extra_info=extra_info)
+    temp_file = PGM_OUTPUT_FILE.with_name(PGM_OUTPUT_FILE.stem + "_temp").with_suffix(".json")
+    with temporary_file_cleanup(temp_file):
```

Comment:
wouldn't python built-in `tempfile` also have worked here?

Reply:
Apparently it's a thing haha